lua/astrocommunity/pack/clojure/clojure.lua

```diff
@@ -0,0 +1,54 @@
+-- Clojure support with Conjure plugin
+local utils = require "astronvim.utils"
+return {
+  -- Clojure Language Server
+  {
+    "williamboman/mason-lspconfig.nvim",
+    opts = function(_, opts) opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, "clojure_lsp") end,
+  },
+  -- Clojure parser
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if opts.ensure_installed ~= "all" then
+        opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, "clojure")
+      end
+    end,
+  },
+  -- Parinfer parens management for Clojure
+  {
+    "gpanders/nvim-parinfer",
+    ft = { "clojure" },
+    config = function()
+      vim.g.parinfer_force_balance = true
+      vim.g.parinfer_comment_chars = ";;"
+    end,
+  },
+  -- Conjure plugin for Clojure REPL
+  {
+    "Olical/conjure",
+    -- load plugin on filetypes
+    ft = { "clojure" },
+    config = function()
+      vim.g.maplocalleader = ","
```

Comment:
```suggestion
```

We shouldn't be overriding `maplocalleader` in a plugin pack

Reply:
I could try use the Conjure Clojure mapping prefix rather than explicitly using maplocalleader

```
vim.g.conjure#mapping#prefix = ","
```

If that isn't an option, then the pack would not enable the key mappings.  If that were the case I would need to document in the readme how to activate mappings by setting a localeader or using the Clojure mapping prefix

Reply:
The conjure mapping prefix approach works, so an update has been pushed